tests/pytests/data/h5270_0000_ir2/h5270_0000_ir2_pds3.lbl

```diff
@@ -115,147 +115,6 @@ RADIANCE_OFFSET              = 0.0 <W*m**-2*sr**-1>
 RADIANCE_SCALING_FACTOR      = 0.0169353 <W*m**-2*sr**-1>
 REFLECTANCE_SCALING_FACTOR   = 0.00215382
 
-/* DATA OBJECT DEFINITIONS */
-Object = IMAGE
-  INTERCHANGE_FORMAT = BINARY
-  LINES              = 400
-  LINE_PREFIX_BYTES  = 68
-  LINE_SAMPLES       = 1288
-  SAMPLE_TYPE        = MSB_INTEGER
```

Comment:
why is all of this deleted?

Reply:
For whatever reason two pds3 labels existed in the file. One with 400 lines and the other with 15088 lines which is the actual number of lines in the image.

Using the real number of lines made testing easier